v2/core/LockupDynamicStreamCreator.sol

```diff
@@ -37,11 +37,18 @@ contract LockupDynamicStreamCreator {
         params.broker = Broker(address(0), ud60x18(0)); // Optional parameter left undefined
 
         // Declare some dummy segments
-        params.segments.push(
-            LockupDynamic.Segment({ amount: amount0, exponent: ud2x18(3.14e18), milestone: block.timestamp + 4 weeks })
-        );
-        params.segments.push(
-            LockupDynamic.Segment({ amount: amount1, exponent: ud2x18(0.5e18), milestone: block.timestamp + 2 years })
+        params.segments = new LockupDynamic.Segment[](2);
+        params.segments[0] = LockupDynamic.Segment({
+            amount: uint128(amount0),
+            exponent: ud2x18(1e18),
+            milestone: uint40(block.timestamp + 4 weeks)
+        });
+        params.segments[1] = (
+            LockupDynamic.Segment({
+                amount: uint128(amount1),
+                exponent: ud2x18(3.14e18),
+                milestone: uint40(block.timestamp + 52 weeks)
+            })
```

Comment:
Good catch. Could you also open a PR upstream to fix this in v2-docs?

https://github.com/sablier-labs/v2-docs/blob/67b9c8f1816e95af06de5a7ebe17ce3daeebb5d3/docs/contracts/v2/guides/create-stream/02-lockup-dynamic.mdx#L138

Reply:
I've fixed them in my recent branch created https://github.com/sablier-labs/v2-docs/blob/30094eb55e613b353ab48324f6f46735b7935f6c/docs/contracts/v2/guides/create-stream/02-lockup-dynamic.mdx#L138

Reply:
Great